tests/unittest_brain_turtle.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 rioj7
+
+from textwrap import dedent
+import unittest
+
+import astroid
+
+class TurtleTest(unittest.TestCase):
+
+    def test_pencolor(self):
+        node = dedent("""
+            import turtle
+            turtle.pencolor
+            """)
+        inferred = next(astroid.extract_node(node).infer())
+        self.assertIsInstance(inferred, astroid.FunctionDef)
+        self.assertEqual(inferred.name, 'pencolor')
+
+    def test_Turtle(self):
+        node = dedent("""
+            import turtle
+            turtle.Turtle()
+            """)
+        inferred = next(astroid.extract_node(node).infer())
+        self.assertIsInstance(inferred, astroid.Instance)
```

Comment:
If i understand well, you are not testing the brain you added but the classical `astroid` behavior (i.e not using the brain).
Am i correct?
If yes, i don't see the point.